JSDemos/Demos/TreeList/ColumnChooser/description.md

```diff
@@ -1 +1,13 @@
-The Column Chooser is&nbsp;an&nbsp;element that allows a&nbsp;user to&nbsp;alter the set of&nbsp;visible columns at&nbsp;runtime. To&nbsp;open the column chooser, click the _&laquo;Column Chooser&raquo;_ button at&nbsp;the top of&nbsp;TreeList. Now, you can drag and drop column headers to&nbsp;and from the TreeList, thus showing or&nbsp;hiding columns. Use the column chooser&rsquo;s search box to&nbsp;quickly find the required column.
\ No newline at end of file
+If you want users to change column visibility at runtime, enable the Column Chooser. Set the **columnChooser**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#enabled) property to **true**.  Note that you can make sure that certain columns always stay visible. To do so, set the **columns[]**.[allowHiding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHiding) property to **false**. 
+
+To open the column chooser, users should click the button in the toolbar above the TreeList. The way users show and hide columns depends on the **columnChooser**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#mode):
```

Comment:
Should "`the **columnChooser**.[mode]`" be "`the **columnChooser** [mode]`" (with a space instead of a period)?

Reply:
No, the dot designates that the property `mode` is inside the `columnChooser` object.

Reply:
Okay, thank you.